tests/e2e/scripts/prow-utilities.sh

```diff
@@ -229,10 +229,11 @@ function create_service() {
   case "$#" in
     '1')
       local swagger_json="${1}"
-      retry -n 3 run ${GCLOUD} endpoints services deploy "${swagger_json}"
+      # b/158767160: workaround servicemanagement service deploy issue.
+      retry -n 3 ${GCLOUD} endpoints services deploy "${swagger_json}"
```

Comment:
we just remove "run", what does the "run" do?

Reply:
It is a wrapper which make the script execution failed once it comes into error. I intended to continue the job even it is failed because I saw the service config was `completed` on endpoints portal, though it was not.

Reply:
Just to confirm, this doesn't fix the issue? I see the tests still fail

Reply:
Nop, the [service enabling](https://b.corp.google.com/issues/158767160#comment4) is failed.